### ISSUE TITLE
Full gain on playing sounds when closing app (DEF-3417)

### ISSUE THREAD

**Issue by britzl** [defold/defold#3424]
(LINK REMOVED)"Something makes sounds replay again at full gain if they play soon enough before engine closes, there is something funky going on.

This gives an unprofessional quality to a game.

This can be reproduced by making a small example where audio plays when you click and then quickly close the game with the x. You should hear the audio anomaly"

**Comment by otsakir** [comment]
@britzl , is the issue still present ?

I tried the following:

1. Created an ogg file about 5 secs long.
2. Put it in a project and bound it with a button click.
3. Set the gain for the sound component at a low value. 0.1.
4. Started the application
5. Click the button to make the ogg start playing
6. Closed the application window pressing X button of the window before the ogg stopped playing.

I expected the volume of the sound to increase (gain -> 1). I noticed no difference though. I also tried with looping enabled. Same results. Btw, i tested on linux only.
